### PR TITLE
Use Acha icons

### PR DESCRIPTION
**Pull request by MiguelWeezardo**

### CODE REVIEW

**Review by nineinchnick — APPROVED** [review]
We should attribute these images to the original repo and drop in the EPL somewhere, as we can't relicense the files: https://github.com/someteam/acha/blob/master/LICENSE

File: sql/git/achievements.sql [hunk 1]
```diff
@@ -33,7 +33,8 @@ SELECT
   min(a.achieved_at) AS first_achieved_at,
   min_by(a.achieved_in, a.achieved_at) AS first_achieved_in,
   slice(array_agg(a.fullname ORDER BY a.num_achieved DESC, a.achieved_at) FILTER (WHERE year(a.achieved_at) = year(current_date)), 1, 3) AS recent_top3,
-  slice(array_agg(a.fullname ORDER BY a.num_achieved DESC, a.achieved_at), 1, 3) AS alltime_top3
+  slice(array_agg(a.fullname ORDER BY a.num_achieved DESC, a.achieved_at), 1, 3) AS alltime_top3,
+  format('<img src="aches/%s.png">', acha.id) AS icon
```
Comment (nineinchnick) [inline sql/git/achievements.sql:37]:
```suggestion
  format('<img src="aches/%s.png"/>', acha.id) AS icon
```

**Comment by MiguelWeezardo** [comment]
> We should attribute these images to the original repo and drop in the EPL somewhere, as we can't relicense the files: https://github.com/someteam/acha/blob/master/LICENSE

What would be the scope of the license? Does it cover only the implementation (icons) or also the idea behind this or the achievement names/descriptions?

**Comment by nineinchnick** [comment]
> > We should attribute these images to the original repo and drop in the EPL somewhere, as we can't relicense the files: https://github.com/someteam/acha/blob/master/LICENSE
> 
> The commit message references the acha repo, I'm not sure what more we could do in terms of attribution. As for the license, what would be the scope of it? Does it cover only the implementation (icons) or also the idea behind this or the achievement names/descriptions?

Great question, it covers both. If we want to keep things simple, we could relicense the whole repo from APL to EPL. But I don't mind only relicensing the git achievements part to EPL. I don't think we can use dual-licensing, since it's derivative work, and dual-licensing allows consumers to pick one license and this would allow changing the license.

tl;dr we need to drop in the EPL license file into the sql/git and reports/git subdirs and maybe mention this in the README - we need to attribute the original repository anyway.

**Comment by MiguelWeezardo** [comment]
> > > We should attribute these images to the original repo and drop in the EPL somewhere, as we can't relicense the files: https://github.com/someteam/acha/blob/master/LICENSE
> > 
> > 
> > The commit message references the acha repo, I'm not sure what more we could do in terms of attribution. As for the license, what would be the scope of it? Does it cover only the implementation (icons) or also the idea behind this or the achievement names/descriptions?
> 
> Great question, it covers both. If we want to keep things simple, we could relicense the whole repo from APL to EPL. But I don't mind only relicensing the git achievements part to EPL. I don't think we can use dual-licensing, since it's derivative work, and dual-licensing allows consumers to pick one license and this would allow changing the license.
> 
> tl;dr we need to drop in the EPL license file into the sql/git and reports/git subdirs and maybe mention this in the README - we need to attribute the original repository anyway.

Is it enough from a legal standpoint to drop a LICENSE file in a directory for that license to apply to everything inside? Most automated tools assume the LICENSE from the project root applies to the entirety of the project. Maybe we should make git achievements (`trino-git-achievements`?) a separate project, with the LICENSE in the root, and link to it's GitHub pages from `trino-cicd` GitHub pages?
